packages/lab/src/components/Flow/stories/Insights.tsx

```diff
@@ -0,0 +1,26 @@
+import { theme } from "@hitachivantara/uikit-styles";
+import { HvFlowNode } from "../Node/Node";
+
+export const Insights = (props) => {
+  return (
+    <HvFlowNode
```

Comment:
I think we want "Insights Dashboard" for the `title`.
Also, `color` is no longer needed (same for the other nodes).